pipeline/inferrer/aspect_ratio_inferrer/app/main.py

```diff
@@ -0,0 +1,49 @@
+from fastapi import FastAPI, HTTPException
+from weco_datascience import http
+from weco_datascience.batching import BatchExecutionQueue
+from weco_datascience.image import get_image_from_url
+from weco_datascience.logging import get_logger
+from .encoder import Encoder
+
+logger = get_logger(__name__)
+
+
+# initialise API
+logger.info("Starting API")
+app = FastAPI(title="Aspect ratio extractor", description="extracts aspect ratios")
+logger.info("API started, awaiting requests")
+
+encoder = Encoder()
+batch_inferrer_queue = BatchExecutionQueue(encoder, batch_size=8, timeout=1)
+
+
+@app.get("/aspect_ratio/")
+async def main(query_url: str):
+    try:
+        image = await get_image_from_url(query_url)
```

Comment:
Do we validate on URLs?

Reply:
Yeah, `weco_datascience` figures out whether it's a IIIF URL or not, what to do in either case, resizing, throwing errors etc https://github.com/wellcomecollection/data-science/blob/master/weco_datascience/image.py

That said, requests can only come from the inference manager at the moment so we don't expect to see many funky URLs.